activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -152,9 +152,14 @@ def indexes(table_name) # :nodoc:
         end
 
         def table_options(table_name) # :nodoc:
+          options = {}
           if comment = table_comment(table_name)
-            { comment: comment }
+            options[:comment] = comment
           end
+          if supports_native_partitioning? && partition_definition = table_partition_definition(table_name)
+            options[:options] = "PARTITION BY #{partition_definition}"
+          end
+          options.presence
```

Comment:
I think, `presence` is not needed here

Reply:
the idea of using `.presence` here is to preserve the returning of nil if neither of these conditions are met. i didn't chase the other implications of returning an empty hash instead - do you think it's better to introduce that as the default return here?

Reply:
This is a private API and it is not needed, you can just remove the `presence` call.